doc/sources/ovf/example/ubuntu-server.ovf

```diff
@@ -48,6 +48,10 @@
           <Label>Default User's password</Label>
           <Description>If set, the default user's password will be set to this value to allow password based login.  The password will be good for only a single login.  If set to the string 'RANDOM' then a random password will be generated, and written to the console.</Description>
       </Property>
+      <Property ovf:key="network-config" ovf:type="string" ovf:userConfigurable="true">
+          <Label>Encoded network-config</Label>
+          <Description>This field is optional. It will be taken if no network configuration is set in user-data. The value for network-config has to be base64 encoded.</Description>
```

Comment:
Network configuration cannot be set in user-data. Can you just remove that sentence?
```suggestion
          <Description>This field is optional. The value for network-config has to be base64 encoded.</Description>
```